websites/M/Maki.gg/presence.ts

```diff
@@ -1,6 +1,7 @@
 const presence = new Presence({
-    clientId: "563434444321587202"
+  clientId: "563434444321587202"
```

Comment:
Why did you remove the indent here?

Reply:
I think I understood it wrong, I though Codacy had issues with the spacing because it said "
Expected indentation of 4 spaces but found 2." Now the line starts with 4 spaces in front of it, is that wrong?

Reply:
You have just removed the indentation for no reason lol

Reply:
Uhhh in my local repository I didn't remove them at all lol. Let me update my local one